game/scripts/npc/items/item_silver_edge.txt

```diff
@@ -66,7 +66,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_all_stats"                                 "15 18 21 25 28"
+        "bonus_all_stats"                                 "15 19 24 30 38"
```

Comment:
I feel like the attack speed and stats should scale like 
Attack speed "30 35 40 45 50"
Then the stats scale like "15 20 25 30 35" (I have no idea if they ever did but this item isn't a terrible stat item in dota and it shouldn't be in OAA either)